data-raw/northern-ireland-ltla-summary-metrics.R

```diff
@@ -0,0 +1,129 @@
+library(tidyverse)
+library(IMD)
+library(geographr)
+library(compositr)
+library(sf)
+library(ggridges)
+
+ltla <-
+  boundaries_ltla21 |>
+  st_drop_geometry() |>
+  filter(str_detect(ltla21_code, "^N"))
+
+lookup_northern_ireland_ltla <-
```

Comment:
If you inspect your lookup table, you will notice that nothing changes from 2014 to 2021. This means the table is not needed, and you should remove the call to the lookup table below.

This is not a mistake. The lookup table exists because for other nations, there have been changes over the years, and the lookup keeps all nations, regardless of if there have been changes, for completeness.